package.json

```diff
@@ -8,7 +8,7 @@
     "lerna": "lerna",
     "bootstrap": "lerna bootstrap",
     "dev": "lerna run build --stream --parallel",
-    "testonly": "jest",
+    "testonly": "jest --reporters=default --reporters=jest-junit",
```

Comment:
```suggestion
    "testonly": "jest",
```

Reply:
Move it to `"testall"` so that this doesn't run when locally running tests

Reply:
Sure. That makes sense. I have made the change.